config/dev.exs

```diff
@@ -71,5 +71,5 @@ config :devfest_registration_portal, DevfestRegistrationPortal.Repo,
   username: "postgres",
   password: "postgres",
   database: "devfest_registration_portal_dev",
-  hostname: "localhost",
+  hostname: "db",
```

Comment:
No, It's db because we've set up the database on docker as a service, and it's called db. I think localhost is when you are using the db on your machine.